packages/ra-ui-materialui/src/layout/CloseNotificationContext.ts

```diff
@@ -0,0 +1,6 @@
+import { createContext } from 'react';
```

Comment:
I think this context (and the corresponding hook) should be in `ra-core`, just like `useNotify`.